### ISSUE TITLE
Cannot set redirect url/template for Webfulx OAuth2

### ISSUE THREAD

**Issue by bdemers** [spring-projects/spring-security#6482]
### Summary

`ServerHttpSecurity` does NOT have a method to set the OAuth (auth code flow) redirect URI/template.

The equivalent for `HttpSecurity` is:
```java
http.oauth2Login().redirectionEndpoint().baseUri("/some/path/here")
```

It looks like the default patterns are hardcoded.

I was expecting the OAuth2  config to be similar between `HttpSecurity` and `ServerHttpSecurity`

### Version

5.1.4

If I'm just not seeing the correct method to set this let me know!

**Comment by jgrandja** [comment]
@bdemers This configuration option was added just this week via #5598. 

See `OAuth2LoginSpec.authenticationMatcher(ServerWebExchangeMatcher authenticationMatcher)`.

**Note:** This is only available in master which is 5.2.x.

I'll close this issue but we can reopen if you require further assistance.

**Comment by bdemers** [comment]
Sweet! my search-fu failed me :)

Thanks again @jgrandja !
